include/vapor/RayCaster.h

```diff
@@ -54,17 +58,19 @@ class RENDER_API RayCaster : public Renderer
         //              | /
         //              |/
         //            0 --------X
-        float *frontFace, *backFace;     // user coordinates, size == bx * by * 3
```

Comment:
Variable names do not follow our coding convention

Reply:
which one specifically?

Reply:
All of them. Private variable names start with and underscore.

Reply:
This comment is addressed by commit 1fb9c9fd.